### ISSUE TITLE
[Request] Delegate points from one player to another

### ISSUE THREAD

**Issue by Mysmonstret** [Alumian/CEPGP-Retail#182]
When people are on standby, they often go to their alts. The points then get awarded to the alt who is on the standby list (I don't want to give out to people who are offline).

I suggest the ability to give every player on the standby list a "parent" so to speak, that will be awarded all the points that the player on the standby list would've otherwise have got.

This means that someone can go to an alt and get put on the standby list, and the points go to their Main character.

Alternatively there could be an option upon clicking characters in the guild list to tick "Alt character" and then in turn assign another character as that characters benefactor of points. This could also be done by rank maybe, "Assign rank as Alt rank". This will only work for guilds like mine where we only raid on one character of course.

**Comment by Alumian** [comment]
Hi Mysmonstret, why don't you want to award to offline?

**Comment by Mysmonstret** [comment]
It's not an issue to be offline if you are on an alt, but that requires manual checking during the raid if people are on alts or not and I don't really have time for that when raid leading.

Awarding to people just being offline and not on an alt is a no go for me. If you are on standby to get standby EP then you should also be able to come at a moments notice if we need you. Being offline means you could be anywhere really. I trust most of my members, but not all of them, and don't want the system being abused by being on standby and then going offline and leaving your house for free points.

**Comment by Alumian** [comment]
Ah yes I understand. We had the same policy in my guild. The only reason why I ask is because what you're asking for is quite complex, it's actually a few feature requests in one because I haven't written the alt feature just yet. I'll keep this open for the time being, but I do have a pretty nasty backlog of requests I need to get through.

**Comment by Mysmonstret** [comment]
Yes this is just a QoL thing, definately not on any sort of priority list what so ever.

I have asked my members to just whisper me for standby if they are on an alt, and I transfer the points manually after the raid. Works :)
